rma_purchase_analytic/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2017 Eficent Business and IT Consulting Services S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Analytic Account in RMA purchase",
+    "version": "12.0.1.0.0",
+    "author": "Eficent,"
+              "Odoo Community Association (OCA)",
+    "license": "LGPL-3",
+    "website": "http://www.eficent.com",
+    "category": "Analytic",
+    "depends": ["rma_account", "rma_analytic", "purchase_analytic"],
```

Comment:
shouldn't this module depend of `rma_purchase`? :thinking:

Reply:
yes